nemo/nemo/backends/pytorch/actions.py

```diff
@@ -1065,15 +1065,15 @@ def __extract_dynamic_axes(port_name: str, ntype: NeuralType,
             dynamic_axes = None
 
         local_parameters = {}
-        for key, value in module._local_parameters.items():
-            local_parameters[key] = value
+        if local_parameters is not None:
```

Comment:
Do you mean to check if `module._local_parameters` is not None? `local_parameters` is set immediately before as an empty dict, so it shouldn't ever be None.

Reply:
yes, just fixed